pages/send/Send.tsx

```diff
@@ -139,36 +149,15 @@ export function Send() {
           {isScanning && (
             <>
               <FocusableCamera onScanned={handleScanned} />
-              <View className="flex flex-row items-stretch justify-center gap-4 p-6">
-                <Button
-                  onPress={openKeyboard}
-                  variant="secondary"
-                  className="flex flex-col gap-2 flex-1"
-                >
-                  <KeyboardIcon className="text-secondary-foreground" />
-                  <Text>Manual</Text>
-                </Button>
-                <Button
-                  variant="secondary"
-                  className="flex flex-col gap-2"
-                  onPress={() => {
-                    router.push("/send/address-book");
-                  }}
-                >
-                  <BookUser className="text-secondary-foreground" />
-                  <Text>Contacts</Text>
-                </Button>
-                <Button
-                  onPress={paste}
-                  variant="secondary"
-                  className="flex flex-col gap-2 flex-1"
-                >
-                  <ClipboardPaste className="text-secondary-foreground" />
-                  <Text>Paste</Text>
-                </Button>
-              </View>
             </>
           )}
+          {!isScanning && !keyboardOpen && permissionStatus === PermissionStatus.DENIED &&
+            <View className="flex-1 h-full flex flex-col items-center justify-center gap-2 p-6">
+              <CameraIcon className="text-foreground" size={72} />
+              <Text className="text-2xl text-foreground text-center">Camera Permissions Denied</Text>
+              <Text className="text-muted-foreground text-xl text-center">It seems you denied permissions to use your camera. You might need to go to your operating system settings to allow access to your camera again.</Text>
```

Comment:
```suggestion
              <Text className="text-muted-foreground text-xl text-center">It seems you denied permissions to use your camera. You might need to go to your device settings to allow access to your camera again.</Text>
```

We can add a button to directly open the settings: https://stackoverflow.com/questions/60690479/how-to-open-app-settings-page-in-react-native-android